scripts/release-post.js

```diff
@@ -32,16 +32,21 @@ function sendRequest (opts) {
     const options = Object.assign({
       headers: { 'User-Agent': 'nodejs.org release blog post script' }
     }, opts)
-
-    request(options, (err, res, body) => {
+    fetch(options.url, options).then(resp => {
+      if (resp.status !== 200) {
+        return reject(new Error(`Invalid status code (!= 200) while retrieving ${options.url}: ${resp.status}`))
+      }
+      if (options.json) {
+        resp.json().then(json => resolve(json))
+      }
+      resp.text().then(text => resolve(text))
+    }).catch(err => {
+      if (err.status !== 200) {
```

Comment:
Please remove this `if` statement. This will always evaluates to `true` because afaik there is no status `property` on [`node-fetch errors`](https://github.com/bitinn/node-fetch/blob/master/ERROR-HANDLING.md) and errors thrown in above suggestions.